app/bundles/StageBundle/Helper/StageHelper.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Mautic\StageBundle\Helper;
+
+use Mautic\LeadBundle\Entity\Lead;
+use Mautic\LeadBundle\Model\LeadModel;
+use Mautic\StageBundle\Entity\Stage;
+use Mautic\StageBundle\Model\StageModel;
+use Psr\Log\LoggerInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class StageHelper
+{
+    protected LeadModel $leadModel;
+
+    protected StageModel $stageModel;
+
+    private LoggerInterface $logger;
+
+    private TranslatorInterface $translator;
+
+    public function __construct(
+        LeadModel $leadModel,
+        StageModel $stageModel,
+        LoggerInterface $logger,
+        TranslatorInterface $translator
+        ) {
+        $this->leadModel  = $leadModel;
+        $this->stageModel = $stageModel;
+        $this->logger     = $logger;
+        $this->translator = $translator;
+    }
```

Comment:
Why are 2 properties protected?
```suggestion
    public function __construct(
        private LeadModel $leadModel,
        private StageModel $stageModel,
        private LoggerInterface $logger,
        private TranslatorInterface $translator
    )
    {
    }
```